services/ymax-planner/src/engine.ts

```diff
@@ -186,6 +183,10 @@ type ProcessPortfolioPowers = Pick<
   depositBrand: Brand<'nat'>;
   feeBrand: Brand<'nat'>;
   portfolioKeyForDepositAddr: Map<Bech32Address, string>;
+  vstoragePathPrefixes: {
+    PORTFOLIOS_PATH_PREFIX: string;
+    PENDING_TX_PATH_PREFIX: string;
```

Comment:
I suggest not using ALL_CAPS now that it's not a constant.

Reply:
I agree with @dckc; PORTFOLIOS_PATH_PREFIX and PENDING_TX_PATH_PREFIX are no longer constants.
```suggestion
    portfoliosPathPrefix: string;
    pendingTxPathPrefix: string;
```

Reply:
Done in #12063.